meteor/lib/jsonSchemaUtil.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * The custom JSONSchema properties we can use for building the UI
+ */
+export enum SchemaFormUIField {
```

Comment:
Shouldn't this live in `shared-lib` or something? Feels like a pretty system-wide set of constants.